### ISSUE TITLE
about diffusion training

### ISSUE THREAD

**Issue by Bobo-y** [moojink/openvla-oft#18]
thanks for your great work! I used use_l1_regression to reproduce the simulation resultsof libero, but when I used use_diffusion, the loss converged slowly and fluctuated greatly. Is there anything special that I need to pay attention to in diffusion training?

**Comment by Bobo-y** [comment]
use grad_acc  is work

**Comment by LanrenzzzZ** [comment]
> thanks for your great work! I used use_l1_regression to reproduce the simulation resultsof libero, but when I used use_diffusion, the loss converged slowly and fluctuated greatly. Is there anything special that I need to pay attention to in diffusion training?


Hello, I have the same question. Can you tell me how to solve it?

**Comment by BBBBBBob** [comment]
Hi, can you share your finetuning hyperparameters?
